wranglertools/import_data.py

```diff
@@ -1663,10 +1698,17 @@ def main():  # pragma: no cover
         sys.exit(1)
     # establish connection and run checks
     connection = FDN_Connection(key)
-    cabin_cross_check(connection, args.patchall, args.update, args.infile,
-                      args.remote, args.lab, args.award)
-    # support for xlsx only - adjust if allowing different
-    workbook, sheetnames = digest_xlsx(args.infile)
+    # support for xlsx and google sheet url or sheets id
+    inputname, booktype = check_and_return_input_type(args.infile)
+
+    cabin_cross_check(connection, args.patchall, args.update, args.remote, args.lab, args.award)
+
+    # need to google authenticate to allow gsheet to be read
+    gauth = None
+    if booktype == 'gsheet':
+        gauth = google_authenticate()
```

Comment:
As mentioned before it's highly probable you will want to provide alternative means of specifying the Google credentials, either by argument directly or environment variable to path. I don't see how a user installing from `pip` could easily use this. You'll need at least some barebones documentation on this as well.